tools/peptideshaker/peptide_shaker.xml

```diff
@@ -16,14 +16,18 @@
         #set $samp_str = "Sample_%s" % datetime.now().strftime("%Y%m%d%H%M%s")
         #set $temp_stderr = "peptideshaker_stderr"
 
+
         mkdir output_reports;
         cwd=`pwd`;
 
```

Comment:
I would rather use Python magic than Bash magic. Simply because I think this is more readable.
```
#import os
#set $PEPTIDE_SHAKER_ROOT_PATH = os.path.dirname( os.environ['PEPTIDESHAKER_JAR_PATH'] )
#set $JAR_NAME = os.path.basename(os.environ['PEPTIDESHAKER_JAR_PATH'])
#set $tmp_peptideshaker_jar_path = os.path.join( $PEPTIDE_SHAKER_ROOT_PATH,  $JAR_NAME)
```

```java -cp $tmp_peptideshaker_jar_path```